src/App.jsx

```diff
@@ -1,107 +1,31 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function hasCommens(commensId) {
+  return commentsFromServer.filter(commens => commens.postId === commensId)
+    || null;
+}
```

Comment:
this method will never return null since filter always returns array which is true in context of boolean

Reply:
also, it is better to change the function name, since this name is good for boolean. but your function returns an array of comments. consider to chose something like `getPostComments`. And the param of the function is basically post id not comments